packages/patternfly-4/react-core/src/components/Nav/NavList.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { NavVariants } from './NavVariants';
+import styles from '@patternfly/patternfly/components/Nav/nav.css';
+import { css } from '@patternfly/react-styles';
+
+import {OneOf} from '../../helpers/typeUtils';
+
+export interface NavListProps extends React.DetailedHTMLProps<React.HTMLAttributes<HTMLUListElement>, HTMLUListElement> {
+  /** Children nodes */
+  children?: React.ReactNode;
+  /** Additional classes added to the list */
+  className?: string;
+  /** Indicates the list type. */
+  variant?: OneOf<typeof NavVariants, keyof typeof NavVariants>;
```

Comment:
I think just `NavVariants` will do. If not, use `NavVariants | keyof typeof NavVariants`.